app/services/transaction_search/searcher.rb

```diff
@@ -9,10 +9,10 @@ class Searcher
     # to SearchForm) that `register` handles.
     cattr_accessor(:default_searchers) do
       [
-        TransactionSearch::FacilitySearcher,
         TransactionSearch::AccountSearcher,
         TransactionSearch::ProductSearcher,
         TransactionSearch::AccountOwnerSearcher,
+        # TransactionSearch::OrderedForSearcher,
```

Comment:
Is this intentionally left commented out?

Reply:
yeah I had an error that I hadn't gotten to yet